torch/csrc/jit/codegen/cuda/scheduler/vectorize_helper.cpp

```diff
@@ -230,22 +240,206 @@ ContiguousInnerDimensionsMapper::ContiguousInnerDimensionsMapper(
         // Initiailze the extent for the mapped iter domain
         ProjectedExtent pe;
         pe.multiplyNumeratorValue(commonOrConstExtent(ca_map_, id));
-        projected_extent_[id] = pe;
+        addProjectedExtent(id, pe);
       } else if (!id->isBroadcast()) {
         // Ignore broadcasts in the reference. Otherwise, remove non-contiguous
         // IDs in the reference tensor as this is the contiguous mapper.
         reordered_root.clear();
       }
     }
-    projected_root_ids_[reference] = reordered_root;
-    // Project reference IDs to rfactor if necessary, otherwise function will
-    // just pass through root to rfactor.
-    projectIdToRFactor(reference, reordered_root);
+    reference_information = MappedDomain::build(
+        reordered_root, projectIdToRFactor(reference, reordered_root));
+  }
+  // Stop recording before traversal
+  recording_ = false;
+
+  // Set MaxInfoSpanningTree::reference_info_ before traversal
+  reference_info_=reference_information;
+  // Set ContiguousInnerDimensionsMapper::tv_infos_ entry for reference
+  tv_infos_[reference] = reference_information;
+
+  traverse(this);
+}
+
+ContiguousInnerDimensionsMapper ContiguousInnerDimensionsMapper::map(
+    TensorView* reference,
+    const std::vector<IterDomain*>& reference_ids,
+    std::shared_ptr<const ComputeAtMap> ca_map,
+    const std::unordered_set<Split*>& divisible_splits) {
+
+  return ContiguousInnerDimensionsMapper(
+      reference, reference_ids, ca_map, divisible_splits);
+}
+
+void ContiguousInnerDimensionsMapper::propagateExtentSplitBackward(
```

Comment:
Just pulled this code into their own functions. Shouldn't be anything but code movement.